features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2782,6 +2785,10 @@
                 <BasePath>/console/</BasePath>
             </Context>
         </WebApp>
+        <Servlet>
+            <Context>/commonauth(.*)</Context>
```

Comment:
In addition to the added default servlets, let's template this part to add new servlets via deployment.toml

Reply:
Fixed by - https://github.com/wso2/carbon-identity-framework/pull/4239/commits/6e938ab0ef0d21451a47955f58b1813b7bc8ce2b